src/ansi-c/library/string.c

```diff
@@ -500,7 +516,7 @@ inline char *strdup(const char *str)
 
 #undef memcpy
 
-inline void *memcpy(void *dst, const void *src, size_t n)
+void *memcpy(void *dst, const void *src, size_t n)
```

Comment:
The problem is that we don't have frames for inlined copies of functions. Thus goto-symex cannot remove local variables when leaving the inlined function scope. Consequently we get type conflicts because we have multiple copies of an array of dynamic size with different size in the same frame.